cluster/saltbase/salt/kubelet/default

```diff
@@ -3,23 +3,10 @@
   {% set daemon_args = "" -%}
 {% endif -%}
 
-{% if grains.api_servers is defined -%}
-  {% set api_servers = "--api-servers=https://" + grains.api_servers -%}
-{% elif grains.apiservers is defined -%} # TODO(remove after 0.16.0): Deprecated form
-  {% set api_servers = "--api-servers=https://" + grains.apiservers -%}
-{% elif grains['roles'][0] == 'kubernetes-master' -%}
```

Comment:
I don't really know what to do about that other than some ugly "rewrite the master kubelet's kubeconfig file from here" magic. I lean towards saying "--api-servers is past its deprecation period. This code path only configured --api-servers for you; as soon as that was deprecated so was this code path." Ofc the reality is that these people probably never saw the deprecation warning... 

It must be pretty dangerous to rely on this salt setup and your own external provisioning scripts without testing, right? How many people do you think have really tried this without experiencing constant breakage?